profiles/common/modules/features/multisite_translations/multisite_translations.module

```diff
@@ -109,29 +124,21 @@ function multisite_translations_views_query_alter(&$view, &$query) {
   }
 }
 
-/*
- * hook_init
- */
-/*
-function multisite_translations_init() {
-  create_translation_roles();
-}
```

Comment:
OMG good that you catched this terrible mistake. I guess this was probably put in to debug something when this was originally developed, and then forgotten. Very good job spotting it!